### PR TITLE
Update rose-pine.theme.css

### PR DESCRIPTION
**Pull request by Papa-Quill**
Fix all the colors that haven't been maintained

### CODE REVIEW

File: rose-pine.theme.css [hunk 1]
```diff
@@ -64,20 +59,39 @@
...
+}
+
+.avatar-2e8lTP {
+    background-color: #2a273f !important;
+    border-style: solid !important;
+    border: var(--bg-overlay-3,var(--channeltextarea-background)) !important;
+}
+
+body {
+    --font-display: Fira Code;
```
Comment (mvllow) [inline rose-pine.theme.css:77]: This may be better in `:root` and with a fallback font set to monospace.

File: rose-pine.theme.css [hunk 2]
```diff
@@ -64,20 +59,39 @@
...
+}
+
 .body-2wLx-E, .headerTop-3GPUSF, .bodyInnerWrapper-2bQs1k, .footer-3naVBw {
     background-color: var(--background-tertiary);
 }
 
 .title-17SveM, .name-3Uvkvr{
-    font-size: var(--Chat-Font-Size);
+    font-size: 14px;
```
Comment (mvllow) [inline rose-pine.theme.css:85]: This was a shared variable before, not sure if it's necessary to change away from that. I would, however, be find with having different size variables in the `:root` for easy change in the future, e.g. `--font-size-title: 14px;` and `--font-size-username: 14px;`

File: rose-pine.theme.css [hunk 3]
```diff
@@ -64,20 +59,39 @@
...
+}
+
+.avatar-2e8lTP.clickable-31pE3P {
+    border-style: solid !important;
+}
+
+.avatar-2e8lTP {
+    background-color: #2a273f !important;
+    border-style: solid !important;
+    border: var(--bg-overlay-3,var(--channeltextarea-background)) !important;
```
Comment (mvllow) [inline rose-pine.theme.css:73]: I don't see `--bg-overlay-3` set anywhere. I believe the `--channeltextarea-background` would be a fallback here so it should work but curious if that's being set somewhere else that I'm not seeing.

**Review by mvllow — CHANGES_REQUESTED** [review]

**Comment by mvllow** [comment]
Also, it may be easier if I merge #16 first—not sure if there's much overlap there or not.

File: rose-pine.theme.css (hunk 3 above)
Comment (Papa-Quill) [reply rose-pine.theme.css]: Removing these changes due to these classes no longer being used

File: rose-pine.theme.css (hunk 2 above)
Comment (Papa-Quill) [reply rose-pine.theme.css]: Removing these changes due to these classes no longer being used

**Review by Papa-Quill — COMMENTED** [review]
Latest commit fixes these issues by removing the unused/leftover code

**Comment by mvllow** [comment]
We'll need to resolve conflicts before review. I see the font was removed but we should leave that and perhaps in the future we can document how the user can change it.

**Review by mvllow — CHANGES_REQUESTED** [review]
Let's leave the font for now and resolve conflicts.

**Comment by mvllow** [comment]
GitHub is still telling me there are conflicts—please try rebasing and that should do the trick

**Comment by mvllow** [comment]
This is the conflict I'm seeing if that helps:

```css
<<<<<<< main

.mainCard__0ec2c {
    background-color: var(--background-secondary) !important;
}

.body-2wLx-E,
.headerTop-3GPUSF,
.bodyInnerWrapper-2bQs1k,
.footer-3naVBw,
.panels-3wFtMD {
    background-color: var(--background-tertiary) !important;
}

.peopleColumn-1wMU14,
.panels-j1Uci_,
.peopleColumn-29fq28,
.peopleList-2VBrVI,
.content-2hZxGK,
.header-1zd7se,
.root-g14mjS .small-23Atuv .fullscreenOnMobile-ixj0e3 {
    background-color: var(--background-secondary) !important;
}

.textArea-12jD-V,
.lookBlank-3eh9lL,
.threadSidebar-1o3BTy,
.scrollableContainer-2NUZem,
.perksModalContentWrapper-3RHugb,
.theme-dark .footer-31IekZ,
.theme-light .footer-31IekZ {
    background-color: var(--background-tertiary) !important;
}

.numberBadge-2s8kKX,
.base-PmTxvP,
.baseShapeRound-1Mm1YW,
.bar-30k2ka,
.unreadMentionsBar-1Bu1dC,
.mention-1f5kbO,
.active-1SSsBb,
.disableButton-220a9y {
    background-color: var(--ping) !important;
}

.lookOutlined-3sRXeN.colorRed-1TFJan {
    color: var(--ping) !important;
}

.header-3OsQeK,
.container-ZMc96U {
    box-shadow: none !important;
    border: none !important;
}

.content-1gYQeQ,
.layout-1qmrhw,
.inputDefault-3FGxgL,
.input-2g-os5,
.input-2z42oC,
.role-2TIOKu,
.searchBar-jGtisZ {
    border-radius: 6px !important;
}

.layout-1qmrhw:hover,
.content-1gYQeQ:hover {
    background-color: var(--background-modifier-selected) !important;
}
=======
>>>>>>> main
```

**Comment by mvllow** [comment]
Thank you ♥

**Pull request merged**